### PR TITLE
Fix assert statements and deprecated logger.warn's

### PR DESCRIPTION
**Pull request by bvavala**
This PR addresses #288  and #484 .

### CODE REVIEW

File: python/pdo/test/storage.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def verify_store_signature(store_response, duration, verifying_key) :
     result = client.store_blocks([block_data], duration=default_duration)
     assert result
 
-    assert verify_store_signature(result, default_duration, client.verifying_key)
+    assert (1 == verify_store_signature(result, default_duration, client.verifying_key))
```
Comment (cmickeyb) [inline python/pdo/test/storage.py:81]: why are we using assert here? it pretty much blows up the entire application. can we just use the existing exceptions? there is already a very nice "SGX error"

File: python/pdo/test/storage.py [hunk 2]
```diff
@@ -109,7 +110,7 @@ def verify_store_signature(store_response, duration, verifying_key) :
     result = client.store_blocks(block_data, duration=default_duration)
     assert result
 
-    assert verify_store_signature(result, default_duration, client.verifying_key)
+    assert (1 == verify_store_signature(result, default_duration, client.verifying_key))
```
Comment (cmickeyb) [inline python/pdo/test/storage.py:114]: same. this is pretty draconian.

**Review by cmickeyb — CHANGES_REQUESTED** [review]
abort should only be used for a programmer error, not an environment error. please replace with one of the standard exceptions.

File: python/pdo/test/storage.py (hunk 1 above)
Comment (cmickeyb) [reply python/pdo/test/storage.py:81]: and what is the difference between the old version and the new version?

**Comment by bvavala** [comment]
> abort should only be used for a programmer error, not an environment error. please replace with one of the standard exceptions.

I have changed the assert's in the file with raises's.

**Review by cmickeyb — APPROVED** [review]

**Review by prakashngit — APPROVED** [review]

**Pull request merged**